packages/client/modules/outcomeCard/components/OutcomeCardAssignMenu/TaskFooterUserAssigneeMenu.tsx

```diff
@@ -25,33 +25,36 @@ const TaskFooterUserAssigneeMenu = (props: Props) => {
   const {area, menuProps, task, viewer} = props
   const {userId, id: taskId} = task
   const {team} = viewer
-  const {teamMembers} = team || {teamMembers: []}
-  const assignees = useMemo(
-    () => teamMembers.filter((teamMember) => teamMember.userId !== userId),
-    [userId, teamMembers]
-  )
+  const {teamMembers}: any = team || {teamMembers: []}
```

Comment:
TypeScript complains if we try to map through `teamMembers` as it says it's readOnly. I tried to cast it to an interface but it wouldn't accept that. I tried a few solutions from [here](https://stackoverflow.com/questions/46634876/how-can-i-change-a-readonly-property-in-typescript) such as `type Writeable<T> = {-readonly [P in keyof T]-?: T[P]}` with no joy.

I'm reluctantly using an `any` at the moment, (TypeScript was previously recognising `assignees` as `any` too) but I'd love to know if there's a better way